app/views/complaints/_complaints_data.html.erb

```diff
@@ -0,0 +1,53 @@
+<% if @complaints.length == 0 %>
+  <section class="usa-site-alert usa-site-alert--info" aria-label="Site alert">
+    <div class="usa-alert">
+      <div class="usa-alert__body">
+        <h3 class="usa-alert__heading">No issues found</h3>
+        <p class="usa-alert__text">
+          Placeholder text for the message, including any mitigation steps we need to share. 
+        </p>
+      </div>
+    </div>
+  </section>
+<% else %>
+  <div class="usa-table-container--scrollable">
+    <table class="usa-table">
+      <thead>
+        <tr>
+          <th scope="col" role="columnheader">Issue #</th>
+          <th
+            data-sortable
+            scope="col"
+            role="columnheader"
+          > Date </th>
+          <th scope="col" role="columnheader"> Status </th>
+          <th scope="col" role="columnheader"> Grantee </th>
+          <th scope="col" role="columnheader"> Issue </th>
+          <th scope="col" role="columnheader"> TTA </th>
+          <th scope="col" role="columnheader"> RAN </th>
+        </tr>
+      </thead>
+      <tbody>
+      <% @complaints.each do |complaint| %>
+        <tr>
+          <th scope="row" role="rowheader"><%= complaint[:id] %></th>
+          <td 
+            data-sort-value="<%= DateTime.parse(complaint[:attributes][:creationDate]).to_i %>"
+          >
+            <%= DateTime.parse(complaint[:attributes][:creationDate]).strftime("%d/%m/%Y") %>
```

Comment:
[suggestion] I'd switch this to `%m/%d/%Y`

Reply:
lol davida what were you thinking